include/kernel.h

```diff
@@ -1543,9 +1543,10 @@ typedef void (*k_timer_stop_t)(struct k_timer *timer);
  * @param expiry_fn Function to invoke each time the timer expires.
  * @param stop_fn   Function to invoke if the timer is stopped while running.
  *
- * @return N/A
+ * @retval 0 on success
```

Comment:
why change it if this function always return 0 ? There is nothing to check here.